docs/virtualenv_guide.md

```diff
@@ -23,6 +23,18 @@ sudo apt-get install python3.6
 brew install python
 brew install python3
 ```
+* <b>Note:</b>
+Since we currently only support up to Python 3.6.X, you may need to brew install this older Python version. If you have had Python 3.6 installed previously, run the following:
```

Comment:
It looks like this command is specifically for python 3.6.5, so I would say, "if you have python 3.6.5 installed previously".